.circleci/src/jobs/notify_android.yml

```diff
@@ -12,13 +12,10 @@ steps:
       at: .
   - restore_environment_variables
   - restore_yarn_cache
-  - run:
-      name: Create github release notes
-      command: echo "export GITHUB_RELEASE_NOTES=$(tools/parse-release-notes parse --android --language en)" >> ${BASH_ENV}
```

Comment:
why did you remove this part?

Reply:
Somehow going the way with storing it in env vars breaks the newlines in the release notes.... I could not figure out why and how to fix it, so I decided to just execute it twice here....